.eslintrc.yml

```diff
@@ -1 +1,31 @@
-extends: 'eslint-config-node-services'
\ No newline at end of file
+extends: 
+  - 'eslint-config-wikimedia/server'
```

Comment:
This should work as just 'wikimedia/server' unless something's broken.

Reply:
+1

Reply:
No, it works with 'wikimedia/server', Going to update that.